hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/EmptyTxnReservedCapacityCounter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.impl.mapstore.writebehind;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+
+final class EmptyTxnReservedCapacityCounter implements TxnReservedCapacityCounter {
+
+    EmptyTxnReservedCapacityCounter() {
+    }
+
+    @Override
+    public void increment(UUID txnId, boolean backup) {
+
```

Comment:
Shall we throw an `IllegalStateException` here or at least put `assert`?

Reply:
no action is expected from this class, it is fine as is.